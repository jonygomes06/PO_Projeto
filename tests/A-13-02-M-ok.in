7
1
2
2
1
2
3
0
4
20
7
2
2
3
n
2
2
2
n
0
5
3
5
2
3
0
0
