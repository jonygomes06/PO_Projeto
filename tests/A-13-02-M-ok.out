Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 2 deve ser devolvida até ao dia 9.
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 3 deve ser devolvida até ao dia 16.
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Introduza número de dias a avançar: Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Introduza o número de utente: Introduza o número da obra: O utente 2 deve pagar uma multa de EUR 25.
O utente deseja pagar multa (s/n)? Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Introduza o número de utente: Introduza o número da obra: O utente 2 deve pagar uma multa de EUR 85.
O utente deseja pagar multa (s/n)? Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 1 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
2 - Darth Vader - darth@dark.force.coAm - NORMAL - SUSPENSO - EUR 85
3 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
4 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: Introduza o número de utente: Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 1 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
2 - Darth Vader - darth@dark.force.coAm - NORMAL - ACTIVO
3 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
4 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 